src/results-main.js

```diff
@@ -215,6 +216,7 @@
         setElectionInfo(getElectionFromResponse(resp));
         $lists.html(html);
         const hasOnly1Contest = (getContestsFromResponse(resp).length == 1);
+        console.log(hasOnly1Contest);
```

Comment:
@srutherfordbcg can you remove this log line please?

Reply:
oops, removed thanks!